docs/layout/clearfix/index.html

```diff
@@ -45,24 +45,24 @@
     <main class="">
       <article class="bg-near-white bt b--black-10 pa3 ph5-ns">
         <h4 class="f4 mv0 fw6 dib mr4">tachyons-clears</h4>
-        <span class="f4 b dib pl0 ml0 mr4">v2.0.4</span>
-        <span class="f4 b dib pl0 ml0 mr4">92 B</span>
+        <span class="f4 b dib pl0 ml0 mr4">v2.2.0</span>
```

Comment:
When I ran the build script it pulled in the latest version of `tachyons-css/clears`